catalog-rest-service/src/main/resources/json/schema/entity/policies/filters.json

```diff
@@ -3,15 +3,25 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Filters",
   "definitions": {
+    "prefix": {
+      "description": "Prefix path of the entity.",
+      "type": "string"
+    },
+    "regex": {
+      "description": "Regex that matches the entity.",
+      "type": "string"
+    },
     "filters": {
```

Comment:
@amiorin @harshach @pmbrull 

I'd like to get some feedback here on the best way to represent a set of filters within the schema.


A lifecycle rule can have multiple filters. 
Each filter is typically a key=value pair. All rules are AND-ed together to match the entity.



I'm exploring other alternatives myself in the meantime, since I'm not fully satisfied with the current schema.

Reply:
Following on what @amiorin commented about adding some sample data, IMO we could merge the changes, open a new PR for the samples and iterate on how the final data structures look like